### PR TITLE
Migrated time-input to CSS modules

### LINKED ISSUE #569: Convert Time Input component to CSS modules
# Issue Description
To better align with modern CSS practices and modern CSS theming capabilities, we want to update this component to use CSS modules and if applicable, CSS custom properties.

## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Expected Behavior
Component styles are converted to use CSS modules. If applicable, convert SCSS variables to CSS custom properties.

## Current Behavior
Component styles use manual name spaced classes and SCSS variables.

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Migrated time-input to CSS modules

Resolves #569 

@cerner/terra

### Deployment Link:
https://terra-core-deployed-pr-590.herokuapp.com/static/#/site/time-input

### CODE REVIEW

**Comment by mjhenkes** [comment]
These conversions look pretty straight forward 👍

File: packages/terra-time-input/src/TimeInput.jsx [hunk 1]
```diff
@@ -259,15 +261,14 @@ class TimeInput extends React.Component {
...
-          className="terra-hidden-time-input"
           type="hidden"
           name={name}
           value={timeValue}
         />
         <Input
           {...inputAttributes}
           ref={(inputRef) => { this.hourInput = inputRef; }}
-          className="terra-TimeInput-hour"
+          className={styles['time-input-hour']}
```
Comment (mhemesath) [inline packages/terra-time-input/src/TimeInput.jsx]: use `cx` here

File: packages/terra-time-input/src/TimeInput.jsx [hunk 2]
```diff
@@ -283,7 +284,7 @@ class TimeInput extends React.Component {
         <Input
           {...inputAttributes}
           ref={(inputRef) => { this.minuteInput = inputRef; }}
-          className="terra-TimeInput-minute"
+          className={styles['time-input-minute']}
```
Comment (mhemesath) [inline packages/terra-time-input/src/TimeInput.jsx]: use `cx` here

File: packages/terra-time-input/src/TimeInput.jsx (hunk 1 above)
Comment (bjankord) [inline packages/terra-time-input/src/TimeInput.jsx]: Updated: https://github.com/cerner/terra-core/pull/590/commits/a8b7d91b66626725ad7a8335f698f9ba4c31134d

File: packages/terra-time-input/src/TimeInput.jsx (hunk 2 above)
Comment (bjankord) [inline packages/terra-time-input/src/TimeInput.jsx]: Updated: https://github.com/cerner/terra-core/pull/590/commits/a8b7d91b66626725ad7a8335f698f9ba4c31134d

**Comment by mmalaker** [comment]
Functional verification completed. Time input examples display and function correctly after conversion to CSS modules.

**Pull request merged**